tests/unittest/bindings/test_executor_bindings.py

```
@@ -102,6 +102,31 @@ def test_executor_with_managed_weights(model_files, model_path):
+        "weight_int32":
+        torch.tensor([[1, 2], [3, 4]], dtype=torch.int32),
+    }
+    if torch.cuda.is_available():
+        managed_weights["weight_fp16_cuda"] = torch.tensor(
+            [[1.0, 2.0], [3.0, 4.0]], dtype=torch.float16, device="cuda")
+
+    executor = trtllm.Executor(engine_buffer, json_config_str,
+                               trtllm.ModelType.DECODER_ONLY, executor_config,
+                               managed_weights)
+
+    assert executor.can_enqueue_requests() == True
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)